docs/src/app/components/pages/customization/themes.jsx

```diff
@@ -30,7 +30,7 @@ const {
   Toggle,
 } = mui;
 
-const {StylePropable, StyleResizable} = Mixins;
```

Comment:
Haha! Honestly when I said "we can also postpone this if we need to until other PRs impacting docs are merged first," yours was the first one that came to mind!

Thanks @alitaheri!